networks/north-america/us/ma/mbta.json

```diff
@@ -0,0 +1,67 @@
+{
+  "public_transport/platform/bus|MBTA Bus Stop": {
+    "countryCodes": {
+        "us": {
+            "ma": true
```

Comment:
This is a clever way to avoid duplicates and make life easier for clients. Do you think we should change the existing brand entries to also structure `countryCodes` as an object? Each country code could be set to just `true` if it applies nationally but to an object if we want to get more granular.

Alternatively, it’d be simpler for contributors (less verbose syntax) if we stick to a flat array of codes that could be [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-2) codes. But that could be more difficult for a client to use Nominatim for, so maybe the build script could transform into this object structure.

Reply:
Looking into it more, I suppose using the combined codes like `ad-07` in an array might be easier for most people. My developer brain wanted to keep the country and subdivision codes separate and hierarchical but it's no big deal to parse them out later.